scripts/functional-tests.sh

```diff
@@ -309,6 +309,27 @@ EOF
     fi
 }
 
+test_model_train() {
+    # TODO: call `ilab model train`
+    if [[ "$(uname)" == Linux ]]; then
+        # real `ilab model train` on Linux produces models/ggml-model-f16.gguf
+        # fake gml-model-f16.gguf the same as merlinite-7b-lab-Q4_K_M.gguf
+        test -f "${ILAB_DATA_DIR}/models/ggml-model-f16.gguf" || \
+            ln -s merlinite-7b-lab-Q4_K_M.gguf "${ILAB_DATA_DIR}/models/ggml-model-f16.gguf"
+    fi
+}
+
+test_model_test() {
+    echo Using the latest of:
+    ls -ltr "${ILAB_DATA_DIR}"/datasets/test_*
```

Comment:
Should this be part of the if Linux below? I think having a `else` with `echo skipping test_model_test test since not running on Linux` might make sense.

Reply:
Moved under the `if`.

Regarding "skipping" I don't find it very useful. This log is interesting only when the test fails. This message won't help to debug. I prefer concise code. I hope tests on "Darwin" will be added soon.